app/src/components/validators.js

```diff
@@ -32,9 +33,9 @@ const models = {
       return true;
     },
 
-    /** @function contentFileType is required */
+    /** @function contentFileType is required and exists as a valid input file type defined in the fileTypes Dictionary */
     contentFileType: value => {
-      return validatorUtils.isNonEmptyString(value);
+      return validatorUtils.isNonEmptyString(value) && value in fileTypes;
```

Comment:
Consider adding a unit test specifically for the "value not in fileTypes" failure condition in the `describe('models.template.contentFileType` suite in validators.models.spec.js

Reply:
Thank you dude.. they were good suggestions. i believe I've corrected that now